cibuildwheel/selector.py

```diff
@@ -99,10 +99,17 @@ def __call__(self, build_id: str) -> bool:
             return False
         if EnableGroup.GraalPy not in self.enable and fnmatch(build_id, "gp*"):
             return False
-        if EnableGroup.PyodidePrerelease not in self.enable and fnmatch(
-            build_id, "cp313-pyodide_*"
-        ):
-            return False
+        # TODO: Re-enable this when we have Pyodide prereleases again (e.g., 0.29.0a1+)
+        # Python 3.13 support became stable in Pyodide 0.28.0, so it no longer needs a prerelease
+        # flag.
+        # Also update Pyodide tests in unit_test/build_selector_test.py accordingly.
+        # When re-enabling, update the pattern to match the experimental Python version in case
+        # it is bumped to Python 3.14 (likely cp314-pyodide_* but could remain as 3.13 as well).
+        # This depends on the CPython version being used in the Pyodide runtime at the time.
+        # if EnableGroup.PyodidePrerelease not in self.enable and fnmatch(
+        #     build_id, "cp313-pyodide_*"
+        # ):
+        #     return False
```

Comment:
I'm not sure what the best way to handle this would be. We don't have a Pyodide 0.29.0 pre-release yet because it's too early to have one. We usually stay one year behind CPython version compatibility, allowing our packages to complete their upgrades. The `cp313-pyodide_wasm32` identifier might stay for Pyodide 0.29, or it might not. The CPython version could even change between Pyodide 0.29 alphas (or 0.30 alphas, for that matter).

We currently say that "If there are pre-releases available for a newer Pyodide version, the `pyodide-prerelease` can be used to include pre-release versions." in the documentation.

We could document there that `cibuildwheel` can disable the `pyodide-prerelease` option when there are no pre-releases available?

Reply:
I'm personally okay with the commenting approach above.

As for the other points, yeah, I can see that it might get a little awkward around alphas, but that's probably okay, that's what alphas are for :) @henryiii were discussing some ideas in #2448 about the possibility of being able to add new identifiers at runtime, which might help? Having said that, I think we're gonna continue to want the python version in the identifier to support the functionality of the [requires-python feature](https://github.com/pypa/cibuildwheel/blob/a6ec930f15689c27998ef75440c06258ae143a33/cibuildwheel/selector.py#L78).